alvr/vulkan_layer/src/lib.rs

```diff
@@ -8,7 +8,9 @@
     clippy::missing_safety_doc,
     clippy::ptr_offset_with_cast,
     clippy::too_many_arguments,
-    clippy::useless_transmute
+    clippy::useless_transmute,
+    clippy::pedantic,
+    clippy::nursery
```

Comment:
this is a bit redundant, because you are not supposed to use the whole pendantic and or nursery groups when doing lint checks. furthermore I would say not to use any nursery lints whatsoever, because by definition they may have broken detection, and they may be controversial and make the code worse (this last point applies also to pedantic)